pkg/controller/migration/convert/ippools.go

```diff
@@ -77,13 +77,21 @@ func handleIPPools(c *components, install *operatorv1.Installation) error {
 				}
 			}
 		} else {
-			if v4pool != nil {
+			// If assign_ipv4="false" then remove any discovered IPv4 pools.
+			// Since we must have an IPv6 pool here, replace the operator
+			// pools with the IPv6 pool.
+			// This is needed because the operator renders IPAM config
+			// based on the presence of initial ippools in the installation CR.
+			ipv6Pool := render.GetIPv6Pool(install.Spec.CalicoNetwork.IPPools)
```

Comment:
Do we need to requery here or couldn't we use `v6pool` from above?

Reply:
Yeah that wasn't right, I fixed that.